x/crosschain/keeper/cctx_gateway_zevm.go

```diff
@@ -23,6 +25,14 @@ func (c CCTXGatewayZEVM) InitiateOutbound(
 	ctx sdk.Context,
 	config InitiateOutboundConfig,
 ) (newCCTXStatus types.CctxStatus, err error) {
+	// abort if CCTX already contains an initial error message from inbound vote msg
+	// the 'insufficient_depositor_fee' is the only case handled for now
+	if strings.Contains(config.CCTX.CctxStatus.ErrorMessage, types.InboundStatus_insufficient_depositor_fee.String()) {
```

Comment:
Not clean.
We should rather add the inboundStatus to the inboundParams object and check with the enum value directly

Reply:
indeed. That is even better

Reply:
I added an extra `InboundStatus status` field to `InboundParams` struct: https://github.com/zeta-chain/node/pull/3358/commits/a0604ef8e5e77ee5253aaa3a3775388b36496a6f